### PR TITLE
Add details about PHP 7.1 support

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) updates the sys requirements to handle the EOL of PHP 7.1..

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html

### CODE REVIEW

File: src/guides/v2.3/install-gde/system-requirements-tech.md [hunk 1]
```diff
@@ -42,14 +42,15 @@ Magento only uses MySQL features compatible with MariaDB. MariaDB may not be com
 
 ## PHP
 
-{:.bs-callout-info}
-All 3rd party libraries now support PHP 7.2.
- If you are interested in participating in Magento Community projects we welcome your help! See our [ZenHub board][] for a full list of outstanding issues.
+{:.bs-callout-warning}
+PHP 7.1 has reached [End of Life](https://www.php.net/supported-versions.php). To maintain PCI compliance, Magento should not be run on unsupported software.
+Installing from Github will no longer work with Magento 2.3.4/PHP 7.1.
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/system-requirements-tech.md]:
```suggestion
Installing from GitHub will no longer work with Magento 2.3.4/PHP 7.1.
```

File: src/guides/v2.3/install-gde/system-requirements-tech.md [hunk 2]
```diff
@@ -42,14 +42,15 @@ Magento only uses MySQL features compatible with MariaDB. MariaDB may not be com
...
+The only way to install 2.3.4 with PHP 7.1.x is via composer from `repo.magento.com`.
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/system-requirements-tech.md]:
```suggestion
The only way to install 2.3.4 with PHP 7.1.x is with Composer.
```

File: src/guides/v2.3/install-gde/system-requirements-tech.md (hunk 1 above)
Comment (dobooth) [reply src/guides/v2.3/install-gde/system-requirements-tech.md]: Shame

**Comment by dobooth** [comment]
running tests

File: src/_includes/install/php-versions-template.md [hunk 3]
```diff
@@ -2,6 +2,10 @@ Supported PHP versions:
 
 {% for version in supported_php_versions %}
 
+{% if version != '~7.1.3' %}
```
Comment (dshevtsov) [inline src/_includes/install/php-versions-template.md]: 🤦‍♂

**Pull request merged**